py2puml/parser.py

```diff
@@ -132,12 +132,15 @@ def parse_type(
 ):
     definition_type_fqdn = f'{definition_type.__module__}.{definition_type.__name__}'
     if definition_type_fqdn not in domain_items_by_fqdn:
-        if issubclass(definition_type, Enum):
-            parse_enum_type(definition_type, definition_type_fqdn, domain_items_by_fqdn)
-        elif getattr(definition_type, '_fields', None) is not None:
-            parse_namedtupled_class(definition_type, definition_type_fqdn, domain_items_by_fqdn)
-        else:
-            parse_class_type(definition_type, definition_type_fqdn, root_module_name, domain_items_by_fqdn, domain_relations)
+        try:
```

Comment:
why do you need this try-catch block? Errors should not be silenced.

Reply:
Yes, I have misunderstanding for this code. I'll get back to original.